dataprovider-retrofit/src/main/java/ch/srg/dataProvider/integrationlayer/request/parameters/IlMediaType.kt

```diff
@@ -7,4 +7,4 @@ import ch.srg.dataProvider.integrationlayer.data.MediaType
  * <p>
  * License information is available from the LICENSE file.
  */
-class ILMediaType(mediaType: MediaType) : IlParam(mediaType.toString().lowercase())
\ No newline at end of file
+class ILMediaType(mediaType: MediaType) : IlParam(mediaType.toString().lowercase())
```

Comment:
⚠️ **[detekt]** <detekt.MatchingDeclarationName> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The file name 'IlMediaType' does not match the name of the single top-level declaration 'ILMediaType'.